charactersheet/charactersheet/templates/dm/encounter_sections/monster_section.tmpl.html

```diff
@@ -0,0 +1,666 @@
+            <div class="col-sm-10">
+              <input type="text"
+                     class="form-control"
+                     placeholder="Gargantuan"
+                     data-bind="textInput: blankMonster().size">
+            </div>
+          </div>
+          <div class="form-group">
+            <label class="col-sm-2 control-label">Type</label>
+              <div class="col-sm-10">
+                <input type="text"
+                       class="form-control"
+                       placeholder="Dragon"
+                       data-bind='value: blankMonster().type'>
+              </div>
+          </div>
+          <div class="form-group">
+            <label class="col-sm-2 control-label">Alignment</label>
+              <div class="col-sm-10">
+                <input type="text"
+                       class="form-control"
+                       placeholder="Chaotic Evil"
+                       data-bind='value: blankMonster().alignment'>
+              </div>
+          </div>
+          <div class="form-group">
+            <label class="col-sm-2 control-label">Armor Class</label>
+              <div class="col-sm-10">
+                <input type="text"
+                       class="form-control"
+                       placeholder="22 (natural armor)"
+                       data-bind='value: blankMonster().armorClass'>
+              </div>
+          </div>
+          <div class="form-group">
+            <label class="col-sm-2 control-label">Hit Points</label>
+              <div class="col-sm-10">
+                <input type="text"
+                       class="form-control"
+                       placeholder="367 (21d20 + 147)"
+                       data-bind='value: blankMonster().hitPoints'>
+              </div>
+          </div>
+          <div class="form-group">
+            <label class="col-sm-2 control-label">Speed</label>
+              <div class="col-sm-10">
+                <input type="text"
+                       class="form-control"
+                       placeholder="40 ft., fly 80 ft., swim 40 ft."
+                       data-bind='value: blankMonster().speed'>
+              </div>
+          </div>
+          <hr/>
+          <table class="table">
+            <thead>
+              <tr>
+                <th width="16%" class="text-center">Strength</th>
+                <th width="16%" class="text-center">Dexterity</th>
+                <th width="16%" class="text-center">Constitution</th>
+                <th width="16%" class="text-center">Intelligence</th>
+                <th width="16%" class="text-center">Wisdom</th>
+                <th width="16%" class="text-center">Charisma</th>
+              </tr>
+            </thead>
+            <tbody>
+              <tr>
+                <td class="text-center">
+                  <input type="number" class="form-control"
+                    data-bind='value: blankMonster().abilityScores()[0].value'>
```

Comment:
For these ability scores, can you create a general method (like with sorting) that returns the ability score for a given string name? I think having this hardcoded array will not only cause us trouble, it's likely to get unreadable overtime.

**Suggestion** Something like this? I'm open to a different method name.
```js
value: blankMonster().abilityScores().scoreWithName('dex')
```